integration/src/main/java/scripts/testscripts/SnapshotDatasetOperations.java

```diff
@@ -188,7 +189,7 @@ private void resultHasDatasetWithRoles(List<Object> datasets) {
         assertThat(dataset, hasEntry(is("id"), is(datasetId.toString())));
         @SuppressWarnings("unchecked")
         List<Object> roles = (List<Object>) dataset.get("roles");
-        assertThat(roles, containsInAnyOrder("steward", "admin"));
+        assertThat(roles, hasItem("steward"));
```

Comment:
I saw the earlier comments but I still don't think I full understand this change. Can you summarize why it's necessary again?

Reply:
So, the "admin" role is granted via having a permission on an admin group on DataRepo - completely tangent to the actual behavior of the Data Catalog, and isn't needed for Data Catalog operations, so we shouldn't test on its existence.